ui/repl.c

```diff
@@ -595,8 +595,21 @@ int main(int argc, char *argv[])
 {
     uv_setup_args(argc, argv); // no-op on Windows
 #else
+
+#if defined(_P64) && defined(JL_DEBUG_BUILD)
```

Comment:
No, this detects wine generally, but since it's not used anywhere else, I want to avoid the warning.